src/User32.Desktop/User32+KEYBDINPUT.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="KEYBDINPUT"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Contains information about a simulated keyboard event.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct KEYBDINPUT
+        {
+            public VirtualKey wVk;
+            public ScanCode wScan;
+            public KEYEVENTF dwFlags;
+            public int time;
+            public UIntPtr dwExtraInfo;
```

Comment:
Rather: given it is decidedly a _pointer_ on the native, this too should be a `void*`. Then add the attribute on the struct and the file so that an `IntPtr` property accessor is automatically generated.